packages/payment-integration-test-framework/src/fixture/pageObject/playwright/ServerSideRender.ts

```diff
@@ -5,6 +5,6 @@ import ejs from 'ejs';
  */
 export class ServerSideRender {
     async renderFile(filePath: string, data?: Record<string, unknown>): Promise<string> {
```

Comment:
Since no more `await`, it should be valid to remove `async`.
```suggestion
    renderFile(filePath: string, data?: Record<string, unknown>): Promise<string> {
```

Reply:
Yeah possibly.